presto-orc/src/main/java/com/facebook/presto/orc/writer/SegmentedSliceBlockBuilder.java

```diff
@@ -147,14 +148,20 @@ public long getSizeInBytes()
         return openSliceOutput.size() + offsetsSizeInBytes + closedSlicesSizeInBytes;
     }
 
+    @Override
+    public OptionalInt fixedSizeInBytesPerPosition()
+    {
+        return OptionalInt.empty();
```

Comment:
similar comment with "// size is variable based on the per element length"

Reply:
Added